src/transaction.rs

```diff
@@ -10,13 +10,13 @@ use std::collections::{HashMap, VecDeque};
 /// Represents a transaction in an EDI document. A transaction is initialized with an ST segment
 /// and ended with an SE segment.
 #[derive(PartialEq, Debug, Serialize, Deserialize)]
-pub struct Transaction<'a, 'b> {
+pub struct Transaction<'a> {
     /// The numeric code which represents the type of transaction.
     #[serde(borrow)]
     pub transaction_code: Cow<'a, str>,
     /// The name of the transaction type in human-readable form.
     #[serde(borrow)]
-    pub transaction_name: &'b str, // not a Cow because it is a reference to a HashMap value
+    pub transaction_name: Cow<'a, str>, // not a Cow because it is a reference to a HashMap value
```

Comment:
The comment here still says that this is not a `Cow`, but it was updated to be one.

Reply:
Sorry I missed this! Should be good now.